src/App.js

```diff
@@ -53,17 +55,87 @@ const App = () => {
     setResults(workout);
   };
 
+  const [removeExercises, setEx] = useState(null);
+  const removeEx = (workout) => {
+    setEx(workout);
+    setTotal((count) => count - 1);
+
+    const exerciseIndex = selectedExercises.findIndex(
+      (i) => i.name === workout.name
+    );
+    selectedExercises.splice(exerciseIndex, 1);
+
+    console.log("Exercise: " + workout.name);
+    console.log("Index: " + exerciseIndex);
+  };
+
+  const toggleDrawer = (anchor, open) => (event) => {
+    if (
+      event.type === "keydown" &&
+      (event.key === "Tab" || event.key === "Shift")
+    ) {
+      return;
+    }
+
+    setDrawer({ ...drawer, [anchor]: open });
+  };
+
+  const [drawer, setDrawer] = React.useState({
```

Comment:
Some commenting around what these values are for, like what do they mean might be useful? top, left, bottom and right are properties commonly used so in this case, I have *no* idea what it is for :)

Reply:
Haha I can add a comment. It gives the Drawer direction properties. So since our drawer opens from the LEFT, the property being referenced is the left. If we wanted to display the drawer on the right, we would use the RIGHT property.

Reply:
Hm! Interesting! I had no idea from when I saw it, so thanks for agreeing to add a comment :)

Reply:
You're welcome :) I will include this in the new PR as well.